test/common_tests.dart

```diff
@@ -1483,6 +1483,68 @@ void runCommonTests(
         });
       });
 
+      group('lastAccessed', () {
+        test('isNowForNewlyCreatedFile', () {
+          File f = fs.file(ns('/foo'))..createSync();
+          expect(new DateTime.now().difference(f.lastAccessedSync()).abs(),
+              lessThan(new Duration(seconds: 2)));
```

Comment:
I was worried about that when I wrote the lastModified variant (from which this was copied).  But I've never seen it flake, and it's been running on Google's internal continuous test bot for a few months.

https://github.com/google/file.dart/blob/master/test/common_tests.dart#L1487

That being said, if you can think of a better way to test, I'd be happy to change it.